scout/adapter/mongo/institute.py

```diff
@@ -163,7 +163,7 @@ def update_institute(
                 alamut_key if alamut_key != "" else None
             )  # allows to reset Alamut key to None
 
-        updates["$set"]["check_show_all_vars"] = check_show_all_vars == "on"
```

Comment:
Perhaps it was this @dnil, but I'm not sure. This way is safer anyway!

Reply:
Yeah, should be better, but still not working on my local. It's probably around there though.. 😬 
Here's a test (https://scout-stage.scilifelab.se/cust002/F0032570/str/variants):

![Screenshot 2022-08-24 at 09 38 36](https://user-images.githubusercontent.com/758570/186359936-9b4ddb6a-c0f6-4713-8094-f03e3d794503.png)

![Screenshot 2022-08-24 at 09 38 45](https://user-images.githubusercontent.com/758570/186359939-67e3a9a3-c376-43ac-a347-5b7bd0a94430.png)

![Screenshot 2022-08-24 at 09 39 47](https://user-images.githubusercontent.com/758570/186359973-a6c0db14-9724-42b9-b42e-44eb43123e78.png)

The goal is to have the two last ones show the same result.. 😸

Reply:
> The goal is to have the two last ones show the same result..

Agreed. I'll check what's wrong!

Reply:
STRs page works as expected now:

<img width="907" alt="image" src="https://user-images.githubusercontent.com/28093618/186377092-d30a89e9-3a28-420e-b745-6bcb8e6834eb.png">